src/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule/TelemetryHttpModuleOptions.cs

```diff
@@ -35,9 +35,9 @@ public TextMapPropagator TextMapPropagator
     }
 
     /// <summary>
-    /// Gets or sets a callback action to be fired when a request is started.
+    /// Gets or sets a callback function to be fired when a request is started.
```

Comment:
Update to explain that this is for the user to create the `Activity`?

Reply:
Do you think that it will be enough db60663d3732bc2efd1aebcecfa0550e73bccd6a?

If no, I am open to include better wording.